packages/liquidator/src/liquidator.js

```diff
@@ -310,79 +311,31 @@ class Liquidator {
           maxTokensToLiquidateWei: maxTokensToLiquidateWei ? maxTokensToLiquidateWei.toString() : null
         });
       }
+      this.logger.debug({
+        at: "Liquidator",
+        message: "Liquidating position",
+        position,
+        inputPrice: scaledPrice.toString(),
+        minLiquidationPrice: this.liquidationMinPrice,
+        maxLiquidationPrice: maxCollateralPerToken.toString(),
+        tokensToLiquidate: tokensToLiquidate.toString()
+      });
+      const logResult = await this.proxyTransactionWrapper.submitLiquidationTransaction(liquidationArgs);
```

Comment:
this is the primary change to this file. previously, the `liquidator` would submit the liquidation inline here. see the code removed 👇 .

After this PR, the liquidator simply calls the `.proxyTransactionWrapper.submitLiquidationTransaction(liquidationArgs)` which handles all logic associated with the actual liquidation transaction.